### DISCUSSION TITLE
Tracking only in a specific directory

### DISCUSSION THREAD

**Discussion by tinalin5501** [git-lfs/git-lfs#4839] [Q&A] [answered]
Is there a way to only git-lfs to track certain files in certain directories of my git repo? For example, I want track csv files only in the data file, but have csv files untracked when it is not in the data file.

**Comment by bk2204** [comment, marked as answer]
Hey,

Yeah, this is definitely possible.  For example, if you want to track only CSV files in the `data` directory, the pattern is `data/**/*.csv`, which should match CSV files anywhere under `data`.  If you just want the ones directly under `data`, then you can write `data/*.csv`.

Hopefully that helps.

**Reply by Belegos** [reply]
Thank you, this worked for my unreal project, i can upload the characters into lfs by directory name. 

I used "Content/Characters/**/*.uasset filter=lfs diff=lfs merge=lfs -text"

works fine for me 👍
